backend/database/migrations/000002_create_activity_table.up.sql

```diff
@@ -0,0 +1,13 @@
+BEGIN 
+
+CREATE TABLE IF NOT EXISTS activity_ (
+    id        SERIAL PRIMARY KEY,
+    Progress  FLOAT,
+    goal_id   INT,
+    CONSTRAINT goal_fk
```

Comment:
Goal_fk is just name for the foreign key constraint. Each constraint needs a distinct identifier so that it can be referenced and managed separately within the schema.

Reply:
ahhhhh that makes sense, thank you for explaining!